src/widgets/accountRecovery/accountRecoveryWidget.tsx

```diff
@@ -0,0 +1,217 @@
+import React from 'react';
+
+import { parseQueryString } from '../../helpers/queryString'
+
+import { Alternative, Heading, Info, Link, Intro, Separator } from '../../components/miscComponent';
+import { createMultiViewWidget } from '../../components/widget/widget';
+import { useReachfive } from '../../contexts/reachfive';
+import { useRouting } from '../../contexts/routing';
+import { useI18n } from '../../contexts/i18n';
+import { createForm } from '../../components/form/formComponent'
+import { PasswordEditorForm, PasswordEditorFormData } from '../passwordEditor/passwordEditorWidget.tsx'
+import { ReactComponent as Passkeys } from '../../icons/passkeys.svg'
+import styled from 'styled-components'
+
+
+interface MainViewProps {
+
+    /**
+     * Allow an end-user to create a password instead of a Passkey
+     */
+    allowCreatePassword?: boolean
```

Comment:
```suggestion
    /**
     * Allow an end-user to create a password instead of a Passkey
     * @default true
     */
    allowCreatePassword?: boolean
```